src/libNode/Node.cpp

```diff
@@ -1165,6 +1166,17 @@ bool Node::ProcessTxnPacketFromLookup([[gnu::unused]] const bytes& message,
 
   LOG_GENERAL(INFO, "Received from " << from);
 
+  // Avoid using the original message for broadcasting in case it contains
+  // excess data beyond the TxnPacket
```

Comment:
Can you give me some backgroudn on why is it bothering us, any issue caused by not doing so?